make/testing_flags.mk

```diff
@@ -1,4 +1,4 @@
-DEV_TAGS = dev
+DEV_TAGS = dev chainrpc walletrpc signrpc invoicesrpc autopilotrpc watchtowerrpc 
```

Comment:
wouldn't this mean there is now no way to disable them?

Reply:
do we want to disable them in dev builds?

Reply:
how can I as a developer ensure that thing compile when they're turned off?

Reply:
@bhandras how would you like to proceed here? add a new make target?

Reply:
PTAL